nautobot/core/settings.py

```diff
@@ -685,3 +685,20 @@
 
 # Dont load the 'taggit' app, since we have our own custom `Tag` and `TaggedItem` models
 SHELL_PLUS_DONT_LOAD = ["taggit"]
+
+#
+# UI settings
+#
+
+
+# UI_RACK_VIEW_TRUNCATE_FUNCTION
+def UI_RACK_VIEW_TRUNCATE_FUNCTION(device_display_name):
```

Comment:
`flake8` doesn't like defining lambda functions as constants as original approach (listed as an anti pattern).

Reply:
Attempted this but for some reason Django failed to load complaining about a missing config.